test/integration/scheduler_perf/config.yaml

```diff
@@ -0,0 +1,5 @@
+nodeaffinity:
```

Comment:
Where is this being used / what is it for / can we add the yaml at a later date ?  I'd prefer not to have to maintain this so early in the iteration cycle before we have the finalized design unless others disagree.